sqlalchemy_utc/now.py

```diff
@@ -7,6 +7,7 @@
 
 class utcnow(FunctionElement):
     """UTCNOW() expression for multiple dialects."""
+    inherit_cache = True
```

Comment:
Trivial but for stylistic consistency:

```suggestion

    inherit_cache = True
```